bundle/tests/scorecard/kuttl/semeru/01-assert.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+timeout: 60
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: my-wsliberty-app-semeru-compiler-1
+---
+kind: Service
+apiVersion: v1
+metadata:
+  name: my-wsliberty-app-semeru-compiler-1
```

Comment:
It'll be good to ensure that sessionAffinity is set to ClientIP, port and protocol values.